crypto/kzg/src/lib.rs

```diff
@@ -13,13 +13,22 @@ pub use c_kzg::{
     Blob, Bytes32, Bytes48, KzgSettings, BYTES_PER_BLOB, BYTES_PER_COMMITMENT,
     BYTES_PER_FIELD_ELEMENT, BYTES_PER_PROOF, FIELD_ELEMENTS_PER_BLOB,
 };
-pub use c_kzg::{Cell, CELLS_PER_EXT_BLOB};
 use mockall::automock;
 
+pub use eip7594::{
+    constants::{BYTES_PER_CELL, CELLS_PER_EXT_BLOB},
+    Cell, CellID, CellRef, TrustedSetup as PeerDASTrustedSetup,
```

Comment:
Will we have to bake 2 trusted setups for now? AFAIR ckzg peerdas needed a different representation of its points

Reply:
Yep thats correct, peerdasKZG will accept the official trusted setup json file from the consensus-specs